radixdlt/babylon-wallet-android
# Display specific message when receiving a 429 rate limit error

## Description
In this PR the gateway exception becomes part of the `RadixWalletException`. When performing a request, an error can be either an error returned from GW `ServiceError` or an error thrown by the wallet `ClientError`. In order to achieve that I removed the last usages of the old `execute` method and replaced with the `toResult` one wherever needed. Also added a new string for the 429 error.

**NOTE**
I will change the base branch of the refactor to main when #732 is merged

## How to test
* Perform normal usage of the app. It is difficult to be rate limited and to see such an error. The goal is to smoke test the app to see that the errors thrown from GW are shown as they used to.

## PR submission checklist
